WalletWasabi.Fluent/Controls/Spectrum/SpectrumControl.cs

```diff
@@ -40,6 +41,7 @@ public SpectrumControl()
 		_sources = new SpectrumDataSource[] { _auraSpectrumDataSource, _splashEffectDataSource };
 
 		_lineBrush = SolidColorBrush.Parse("#97D234").ToImmutable();
+		_linePen = new(_lineBrush);
```

Comment:
Just a question, why was this added?

Reply:
`_linePen` didn't have a value, when exiting the ctor.